src/WebJobs.Extensions.NotificationHubs/Config/NotificationHubsConfiguration.cs

```diff
@@ -55,8 +43,34 @@ public void Initialize(ExtensionConfigContext context)
             }
             var converterManager = context.Config.GetService<IConverterManager>();
             converterManager = converterManager.AddNotificationHubConverters();
-            var provider = new NotificationHubAttributeBindingProvider(context.Config.NameResolver, converterManager, this);
+            var provider = new NotificationHubAttributeBindingProvider(converterManager, this);
             context.Config.RegisterBindingExtension(provider);
         }
+
+        internal static string GetSettingFromConfigOrEnvironment(string key)
```

Comment:
Ideally yeah. But we want to actually do it the 'right' way at the core level, so we're keeping all of this internal for beta. Tracking issue: https://github.com/Azure/azure-webjobs-sdk/issues/653